netbox/dcim/tables/template_code.py

```diff
@@ -109,6 +109,11 @@
 </a>
 """
 
+OUTER_UNIT = """
+{% load helpers %}
+{% if value %}{{ value }} {{ record.outer_unit }}{% else %}{{ ''|placeholder }}{% endif %}
```

Comment:
We can simplify this if `columns.TemplateColumn` is used:
```suggestion
{% if value %}{{ value }} {{ record.outer_unit }}{% endif %}
```